content/en/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md

```diff
@@ -9,6 +9,14 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.28"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
```

Comment:
Adding `toVersion` to the previous stage would enhance consistency with other feature gate files.

```suggestion
    toVersion: "1.30"
  - stage: beta
    defaultValue: true
    fromVersion: "1.31"
```